homework/shelkovich_ilya/powerCalculator/src/main/java/com/tutrit/java/quickstart/PowerCalculator.java

```diff
@@ -0,0 +1,19 @@
+package com.tutrit.java.quickstart;
+
+import com.tutrit.java.quickstart.exceptions.NegativeNumberException;
+import com.tutrit.java.quickstart.exceptions.ZeroException;
+
+public class PowerCalculator {
+
+    public int calculate(int number, int power) throws NegativeNumberException, ZeroException {
+        if (number < 0 || power < 0) {
+            throw new NegativeNumberException("Number or power is negative!");
```

Comment:
So number or power? you could pass values to the message to make output more verbose, like this:
```
new NegativeNumberException(String.format("Negative valued are not allowed! number is %s, power is %s", number, power));

Reply:
Thx